API.yaml

```diff
@@ -1877,31 +2109,6 @@ components:
             type: array
             items:
               $ref: '#/components/schemas/MarkerSet'
-    DataProvider:
-      type: object
-      properties:
-        name:
-          type: string
-          description: The human readable name
-        id:
-          type: string
-          description: The output provider's ID
-        type:
-          type: string
-          description: "Type of data returned by this output. Serves as a hint to\
-            \ determine what kind of view should be used for this output (ex. XY,\
-            \ Time Graph, Table, etc..). Providers of type TREE_TIME_XY and TIME_GRAPH\
-            \ can be grouped under the same time axis. Providers of type DATA_TREE\
-            \ only provide a tree with columns and don't have any XY nor time graph\
-            \ data associated with it."
-          enum:
-          - TABLE
-          - TREE_TIME_XY
-          - TIME_GRAPH
-          - DATA_TREE
-        description:
-          type: string
-          description: Describes the output provider's features
```

Comment:
I don't remember, was there a trick to avoid random reordering in this file?

Reply:
It's manual to avoid re-ordering. I'll fix it.